internal/gps/cmd.go

```diff
@@ -196,12 +198,31 @@ func runFromRepoDir(ctx context.Context, repo vcs.Repo, timeout time.Duration, c
 	return c.combinedOutput(ctx)
 }
 
+func getenvDuration(key string, def time.Duration) time.Duration {
+	seconds, err := strconv.Atoi(os.Getenv(key))
+
+	if err != nil {
+		return def
+	}
+
+	return time.Duration(seconds) * time.Second
+}
+
 const (
+	// envExpensiveCmdTimeout is the environment variable to read for the
+	// expensiveCmdTimeout default, in seconds.
+	envExpensiveCmdTimeout = "DEP_EXPENSIVE_CMD_TIMEOUT"
```

Comment:
Currently dep isn't relying upon any environment variables (other than in tests). I'm not sure if we want to start doing that, instead of continuing to use flags? I'll let @sdboyer have the final say on that though.

If we do stick with environment variables, they should be documented in the help text too.